### ISSUE TITLE
`/nix/var` exists with a different uid (1000) than planned (0)

### ISSUE THREAD

**Issue by 0ussamaBernou** [DeterminateSystems/nix-installer#1019]
## Error
```
Error: 
   0: Planner error
   1: Error executing action
   2: Action `provision_nix` errored
   3: Action `create_nix_tree` errored
   4: Action `create_directory` errored
   5: `/nix/var` exists with a different uid (1000) than planned (0), consider updating it with `chown 0 /nix/var`
```

## Metadata
|key|value|
|--|--|
|**version**|0.19.0|
|**os**|linux|
|**arch**|x86_64|

**Comment by cole-h** [comment]
If you run the suggested command, are you able to proceed further?

**Comment by 0ussamaBernou** [comment]
kind of after doing `chown 0 /nix/var` it gave the same error for a lot of sub directories of `/nix/var` , it's when i changed ownership recursively `chown -R 0 /nix/var` and it worked.

**Comment by 0ussamaBernou** [comment]
should i close the issue, now or you are planning on updating the error message?
cause the root of the problem is i already had nix installed with the official installer

**Comment by cole-h** [comment]
Let's keep this open for now -- I'll make a quick PR updating the error message to include the `-R`, for completeness sake. Thanks for the report!

**Comment by cole-h** [comment]
I've added a note that "you may need to use `-R`" available in 0.20.0.
